package/src/components/MessageList/MessageList.tsx

```diff
@@ -783,28 +785,47 @@ const MessageListWithContext = <
     }
   };
 
+  const scrollToIndexFailedRetryCountRef = useRef<number>(0);
```

Comment:
we should reset this to 0 at some point! 

maybe whenever scrollTo() is called